psutil/_psbsd.py

```diff
@@ -455,6 +455,28 @@ def sensors_temperatures():
 
         return ret
 
+    def cpu_freq():
+        ret = []
+        num_cpus = cpu_count_logical()
+        for cpu in range(num_cpus):
+            try:
+                current, available = cext.cpu_frequency(cpu)
```

Comment:
This try/except block is too broad. It should only wrap the `cext.cpu_frequency(cpu)` line. 
Also consider renaming `available` to `minmax` or something.